File: ReverseEachWordInSentence.py
# Reverse Each Word In Sentence
# - Reverse each word in a string `s` while keeping word order.
# - Constraints:
#   - 1 <= len(s) <= 10^4

class Solution:
    def reverseWords(self, s: str) -> str:
        return " ".join(word[::-1] for word in s.split())

File: ReverseEachWordInSentence.cs
// Reverse Each Word In Sentence
// - Reverse each word in a string `s` while keeping word order.

using System;
using System.Linq;

class Solution
{
    public string ReverseWords(string s)
    {
        var words = s.Split(' ', StringSplitOptions.RemoveEmptyEntries);
        var reversedWords = words.Select(word => new string(word.Reverse().ToArray()));
        return string.Join(" ", reversedWords);
    }
}
